partiql-conformance-test-generator/src/schema.rs

```diff
@@ -0,0 +1,42 @@
+/// Conformance test document containing namespaces and/or tests
+pub struct TestDocument {
+    pub(crate) namespaces: Namespaces,
+    pub(crate) test_cases: TestCases,
```

Comment:
Non-blocking for this PR: If not mistaken, this means we can have test cases in the root of a test document which translates to having tests directly under a `Scope`; does it mean that we can have tests that belong to no namespace? or we are considering an implicit root namespace.

Reply:
>If not mistaken, this means we can have test cases in the root of a test document which translates to having tests directly under a Scope

Correct.

> does it mean that we can have tests that belong to no namespace? or we are considering an implicit root namespace.

For now, I'm considering `TestDocument` as a "root"/"top"-level namespace.